erts/emulator/test/exception_SUITE.erl

```diff
@@ -1119,7 +1120,13 @@ error_info(_Config) ->
          {send, [[bad,dest], message]},
          {send, [[bad,dest], message, bad]},
 
+         {send_after, [Eons, self(), message]},
```

Comment:
In #4764 I added in the stdlib error_info_lib so that you can validate which arguments should cause an error and also that the correct description is printed. For example: https://github.com/erlang/otp/pull/4757/files#diff-17817455d8726ece362654904cc419cfe59a2c0e5237bda32e3dd98cf1fd5f5fR3000

Do we want to use that for these tests as well?

Reply:
Yes, I think that is a good idea. We'll better do that in a separate pull request.